### PR TITLE
Test: add retry condition for intermittent test fails

### PR DESCRIPTION
**Pull request by ykim-1**
## 📝 Description

**What does this PR do and why is this change necessary?**

## ✔️ How to Test

**What are the steps to reproduce the issue or verify the changes?**

**How do I run the relevant unit/integration tests?**

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

**Comment by ykim-1** [comment]
make testacc

**Comment by zliang-akamai** [comment]
/acctest sha=2deca5a1df9009072fa11058c7446cb5f024fe04 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=1e19bd4e596460700e44af4e8e7e972423f225c8 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=1b253fcaf7f2ca361063b458613c962c65c795b4 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=1b253fcaf7f2ca361063b458613c962c65c795b4 pkg=linode/...

File: linode/image/resource.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	LinodeImageCreateTimeout = 20 * time.Minute
+	LinodeImageCreateTimeout = 25 * time.Minute
```
Comment (zliang-akamai) [inline linode/image/resource.go:21]:
```suggestion
	LinodeImageCreateTimeout = 30 * time.Minute
```

Let's be a little bit more progressive on this one

**Comment by zliang-akamai** [comment]
/acctest sha=7e96a877995db45e229f2e45f00465ea8e51c1d9 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest 7089a191f1ef30da9a0664943480dc25761e8ce7 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=7089a191f1ef30da9a0664943480dc25761e8ce7 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=ec1336d1748fe541fe1f59ce85f246a18b6a4196 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=3584ec8690f89e07eed76aaead9181b96d052949 pkg=linode/...

**Comment by ykim-1** [comment]
/acctest sha=b2c286516c64a2b83cedfad7aa34de50d7da917d pkg=linode/...

**Review by lgarber-akamai — APPROVED** [review]
Nice work! The flapping test in the `/acctest` command isn't within our control and everything else looks good 👍

**Review by yec-akamai — APPROVED** [review]
Nice work!

**Review by yec-akamai — APPROVED** [review]
Nice work!

**Review by zliang-akamai — APPROVED** [review]

**Comment by zliang-akamai** [comment]
/acctest sha=b2c286516c64a2b83cedfad7aa34de50d7da917d pkg=linode/...

**Pull request merged**